front/src/modules/rollingStock/components/RollingStockCard/index.ts

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
you no longer need this when you have two exports

Reply:
Thanks, fixed !